### PR TITLE
add op bitwise_or [numpy]

### PR DESCRIPTION
**Pull request by Yiyan66**
## Description ##
add op bitwise_or

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at https://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by hgt312** [comment]
@mxnet-label-bot add [Numpy]

File: python/mxnet/ndarray/numpy/_op.py [hunk 1]
```diff
@@ -39,7 +39,7 @@
            'around', 'hypot', 'rad2deg', 'deg2rad', 'unique', 'lcm', 'tril', 'identity', 'take',
            'ldexp', 'vdot', 'inner', 'outer', 'equal', 'not_equal', 'greater', 'less', 'greater_equal', 'less_equal',
            'hsplit', 'rot90', 'einsum', 'true_divide', 'nonzero', 'shares_memory', 'may_share_memory', 'diff', 'resize',
-           'nan_to_num']
+           'nan_to_num','bitwise_or']
```
Comment (hgt312) [inline python/mxnet/ndarray/numpy/_op.py]: One whitespace here.

File: python/mxnet/ndarray/numpy/_op.py [hunk 2]
```diff
@@ -1148,6 +1148,55 @@ def lcm(x1, x2, out=None, **kwargs):
     array([ 0, 20, 20, 60, 20, 20], dtype=int64)
     """
     return _ufunc_helper(x1, x2, _npi.lcm, _np.lcm, _npi.lcm_scalar, None, out)
+    
+@set_module('mxnet.ndarray.numpy')
+@wrap_np_binary_func
+def bitwise_or(x1, x2, out=None, **kwargs):
```
Comment (hgt312) [inline python/mxnet/ndarray/numpy/_op.py]: One more blank line.

File: python/mxnet/numpy/multiarray.py [hunk 3]
```diff
@@ -57,7 +57,7 @@
            'blackman', 'flip', 'around', 'arctan2', 'hypot', 'rad2deg', 'deg2rad', 'unique', 'lcm', 'tril',
            'identity', 'take', 'ldexp', 'vdot', 'inner', 'outer', 'equal', 'not_equal', 'greater', 'less',
            'greater_equal', 'less_equal', 'hsplit', 'rot90', 'einsum', 'true_divide', 'nonzero', 'shares_memory',
-           'may_share_memory', 'diff', 'resize', 'nan_to_num']
+           'may_share_memory', 'diff', 'resize', 'nan_to_num','bitwise_or']
```
Comment (hgt312) [inline python/mxnet/numpy/multiarray.py]: Same here.

File: python/mxnet/numpy/multiarray.py [hunk 4]
```diff
@@ -2749,6 +2749,55 @@ def lcm(x1, x2, out=None, **kwargs):
     """
     return _mx_nd_np.lcm(x1, x2, out=out)
 
+@set_module('mxnet.numpy')
```
Comment (hgt312) [inline python/mxnet/numpy/multiarray.py]: Same here.

File: python/mxnet/symbol/numpy/_symbol.py [hunk 5]
```diff
@@ -41,7 +41,7 @@
            'around', 'hypot', 'rad2deg', 'deg2rad', 'unique', 'lcm', 'tril', 'identity', 'take',
            'ldexp', 'vdot', 'inner', 'outer', 'equal', 'not_equal', 'greater', 'less', 'greater_equal',
            'less_equal', 'hsplit', 'rot90', 'einsum', 'true_divide', 'shares_memory', 'may_share_memory', 'diff',
-           'resize', 'nan_to_num']
+           'resize', 'nan_to_num','bitwise_or']
```
Comment (hgt312) [inline python/mxnet/symbol/numpy/_symbol.py]: Same here.

File: python/mxnet/symbol/numpy/_symbol.py [hunk 6]
```diff
@@ -1209,6 +1209,48 @@ def lcm(x1, x2, out=None, **kwargs):
     gcd : The greatest common divisor
     """
     return _ufunc_helper(x1, x2, _npi.lcm, _np.lcm, _npi.lcm_scalar, None, out)
+    
+@set_module('mxnet.symbol.numpy')
```
Comment (hgt312) [inline python/mxnet/symbol/numpy/_symbol.py]: Same here.

File: python/mxnet/numpy/multiarray.py (hunk 4 above)
Comment (haojin2) [reply python/mxnet/numpy/multiarray.py]: also, one more blank line above.

File: python/mxnet/ndarray/numpy/_op.py (hunk 1 above)
Comment (haojin2) [reply python/mxnet/ndarray/numpy/_op.py]: move `bitwise_or` to after `lcm` in this list, same for all other files.

File: python/mxnet/ndarray/numpy/_op.py [hunk 7]
```diff
@@ -1150,6 +1150,56 @@ def lcm(x1, x2, out=None, **kwargs):
...
+    """
+    Returns the bit-wise OR of two arrays element-wise or the bit-wise OR of
+    the underlying binary representation of the integers in the input arrays
+
+    Parameters
+    ----------
+    *x : array_like
+    Input arrays.
```
Comment (artor1os) [inline python/mxnet/ndarray/numpy/_op.py]: indent

File: src/operator/mshadow_op.h [hunk 8]
```diff
@@ -1327,6 +1329,7 @@ struct lcm : public mxnet_op::tunable {
   }
 };
 
+
```
Comment (haojin2) [inline src/operator/mshadow_op.h]: One less blank line here.

**Comment by haojin2** [comment]
Please address the comments and get rid of the change of tvm submodule.

**Pull request merged**